bluespec/Makefile

```diff
@@ -30,21 +30,36 @@ BSC = bsc
 BLUESPECDIR ?= $(shell which $(BSC) | xargs dirname | xargs dirname)/lib
 
 NB_SYSTEMS ?= 1
+CORE_NUM ?= 2
+
+# RiscyOOO flags
+ifeq ($(NB_SYSTEMS),1)
+CORE_SIZE ?= SMALL
+else
+#BSCFLAGS += -D CORE_MINI
+CORE_SIZE ?= TINY
+endif
 
 TOPFILE = $(CURDIR)/CHERI_BGAS_Top.bsv
 
 # relevant folders
 DE10PROBSVSHELLDIR ?= $(CURDIR)/DE10Pro-bsv-shell
 COREDIR ?= $(CURDIR)/Toooba
-COREW_DIRS = $(COREDIR)/src_Core/Core:$(COREDIR)/src_Verifier/BSV-RVFI-DII:$(COREDIR)/src_Core/CPU:$(COREDIR)/src_Core/ISA:$(COREDIR)/src_Core/PLIC:$(COREDIR)/src_Core/Debug_Module:$(COREDIR)/src_Core/BSV_Additional_Libs:$(COREDIR)/src_Core/RISCY_OOO/procs/RV64G_OOO:$(COREDIR)/src_Core/RISCY_OOO/procs/lib:$(COREDIR)/src_Core/RISCY_OOO/coherence/src:$(COREDIR)/src_Core/RISCY_OOO/fpgautils/lib
-WINDCOREIFCDIR ?= $(COREDIR)/libs/WindCoreInterface
-CHERICAPLIBDIR ?= $(COREDIR)/libs/cheri-cap-lib
-TAGCONTROLLERDIR ?= $(COREDIR)/libs/TagController
+
+#================================================================
+# Parameter settings for MIT RISCY
+
+include $(COREDIR)/builds/Resources/Include_RISCY_Config.mk
+
+#COREW_DIRS = $(COREDIR)/src_Core/Core:$(COREDIR)/src_Verifier/BSV-RVFI-DII:$(COREDIR)/src_Core/CPU:$(COREDIR)/src_Core/ISA:$(COREDIR)/src_Core/PLIC:$(COREDIR)/src_Core/Debug_Module:$(COREDIR)/src_Core/BSV_Additional_Libs:$(COREDIR)/src_Core/RISCY_OOO/procs/RV64G_OOO:$(COREDIR)/src_Core/RISCY_OOO/procs/lib:$(COREDIR)/src_Core/RISCY_OOO/coherence/src:$(COREDIR)/src_Core/RISCY_OOO/fpgautils/lib
```

Comment:
Can you remove these lines as soon as you are certain you do not need them anymore?

Reply:
Looks like I cleaned this up.